src/Data/UsageMetadata.php

```diff
@@ -33,7 +33,7 @@ public static function from(array $attributes): self
     {
         return new self(
             promptTokenCount: $attributes['promptTokenCount'],
-            candidatesTokenCount: $attributes['candidatesTokenCount'],
+            candidatesTokenCount: $attributes['candidatesTokenCount'] ?? null,
```

Comment:
candidatesTokenCount property must be nullable

Reply:
Sorry I missed that 🫣